src/main/java/com/pinterest/secor/consumer/Consumer.java

```diff
@@ -145,6 +146,8 @@ protected boolean consumeNextMessage() {
                 final double DECAY = 0.999;
                 mUnparsableMessages *= DECAY;
             } catch (Throwable e) {
+                mMetricCollector.increment("consumer.message_errors.count", "topic:" + rawMessage.getTopic());
```

Comment:
Can we make 'topic:' as an explicit argument to 'increment' call instead of concat string